spark/v2.3/spark/src/test/java/com/netease/arctic/spark/SparkTestContext.java

```diff
@@ -350,7 +350,7 @@ protected List<Object[]> sql(String query, Object... args) throws RuntimeExcepti
       this.rows = new ArrayList<>();
       return ImmutableList.of();
     }
-    result.show();
+    result.show(false);
```

Comment:
why change this?

Reply:
set show() method 's parameter "truncate" to 'false' to show the whole row instead of a row which will be truncated like this: 
[array01. array02...]